arena-artifacts/charts/job-supervisor/templates/rbac.yaml

```diff
@@ -0,0 +1,112 @@
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: job-supervisor
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "arena.labels" . | nindent 4 }}
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: job-supervisor
+  labels:
+    {{- include "arena.labels" . | nindent 4 }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
```

Comment:
Could you please check the permission of  clusterRole? Do we really need these high-risk permissions?